docs/README.md

```diff
@@ -1,41 +0,0 @@
-## EMR Cost Calculator
```

Comment:
thanks for reminding me of this.
I removed it and push back to master, so please merge back master into your branch.